c2corg_api/models/document.py

```diff
@@ -62,8 +53,7 @@ def redirects_to(self):
             Integer, ForeignKey(schema + '.documents.document_id'),
             nullable=True)
 
-    quality = Column(
-        Enum(name='quality_type', inherit_schema=True, *quality_types))
+    quality = Column(enums.quality_type)
```

Comment:
This was to make sure that the enum is created in the `guidebook` schema. Now this is ensured with:
https://github.com/c2corg/v6_api/pull/219/files/dabe5ff4aef6c7a9b8d09e75cef5acb99a2a63fb#diff-058a5619d0451ccd20f1bd7f3f2b6358L9